roles/gitian/templates/gitian-build.sh

```diff
@@ -172,77 +183,114 @@ fi
+            echo "suite_image_path: ${suite_image_path}"
+
+            if [ ! -f ${suite_image_path} ]; then
+                echo "Image not found for suite ${suite}; calling make-base-vm to build it"
+                ./bin/make-base-vm --lxc --arch amd64 --distro debian --suite ${suite}
+            fi
+
+            echo ""
+	        echo "Compiling ${VERSION} Linux"
+	        echo ""
+
+            ./bin/gbuild -j ${proc} -m ${mem} --commit zcash=${COMMIT} --url zcash=${url} ${suite_dir_path}/gitian-linux.yml
+            ./bin/gsign -p "$signProg" --signer "$SIGNER" --release ${VERSION}_${suite} --destination ${gitian_sigs_repo_path}/ ${suite_dir_path}/gitian-linux.yml
+
+            suite_binaries_dir_path=${zcash_binaries_dir_path}/${VERSION}/${suite}
+            mkdir ${suite_binaries_dir_path}
+
+            mv ${build_dir_path}/out/zcash-*.tar.gz ${build_dir_path}/out/src/zcash-*.tar.gz ${suite_binaries_dir_path}
+
+            popd  # pushd ${gitian_builder_repo_path}
+        done
+
 	fi
-	popd
 
         if [[ $commitFiles = true ]]
         then
 	    # Commit to gitian.sigs repo
             echo ""
             echo "Committing ${VERSION} Signatures"
             echo ""
-            pushd gitian.sigs
-            git add ${VERSION}/${SIGNER}
-            git commit -a -m "Add ${VERSION} signatures for ${SIGNER}"
+            pushd ${gitian_sigs_repo_path}
+            git add ${VERSION}_${suite}/${SIGNER}
+            git commit -a -m "Add ${VERSION}_${suite} signatures for ${SIGNER}"
             popd
         fi
 fi
 
 # Verify the build
 if [[ $verify = true ]]
 then
-	# Linux
-	pushd ./gitian-builder
-	echo ""
-	echo "Verifying ${VERSION} Linux"
-	echo ""
-	./bin/gverify -v -d ../gitian.sigs/ -r ${VERSION} ../zcash/contrib/gitian-descriptors/gitian-linux.yml
-	popd
+    # Linux
+    pushd ${gitian_builder_repo_path}
+
+    for suite in ${suites} ; do
+        echo "processing suite ${suite}"
+
+        suite_dir_path=${suite_descriptors_dir_path}/${suite}
+        echo "suite_dir_path: ${suite_dir_path}"
+
+        echo ""
+        echo "Verifying ${VERSION}_${suite} Linux"
+        echo ""
+
+        ./bin/gverify -v -d ${gitian_sigs_repo_path}/ -r ${VERSION}_${suite} ${suite_dir_path}/gitian-linux.yml
+    done
+
+    popd
 fi
 
 # Sign binaries
 if [[ $sign = true ]]
 then
 
-        pushd ./gitian-builder
+    pushd ${gitian_builder_repo_path}
 	popd
 
-        if [[ $commitFiles = true ]]
-        then
-            # Commit Sigs
-            pushd gitian.sigs
-            echo ""
-            echo "Committing ${VERSION} Signed Binary Signatures"
-            echo ""
-            git commit -a -m "Add ${VERSION} signed binary signatures for ${SIGNER}"
-            popd
-        fi
+    if [[ $commitFiles = true ]]
+    then
+        # Commit Sigs
+        pushd ${gitian_sigs_repo_path}
+        echo ""
+        echo "Committing ${VERSION} Signed Binary Signatures"
```

Comment:
Should this also be `${VERSION}_${suite}`?

Reply:
As currently written, no: after having added signatures to the signatures repository, it is generating a single commit for all the suites it added signatures for.

Reply:
Actually on further inspection, I'm not sure this section is doing anything at all...

Reply:
This section was added in the initial commit for this project, apparently as a placeholder for a "sign binaries" step that was never added.

Maybe I should remove the whole section...

Reply:
Made a new PR for that

Reply:
https://github.com/zcash/zcash-gitian/pull/76